### PR TITLE
Add Cloudrun for Anthos example

### PR DESCRIPTION
**Pull request by upodroid**
If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by upodroid** [comment]
bump

File: mmv1/templates/terraform/examples/cloud_run_anthos.tf.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  manifest = {
+    "apiVersion" = "serving.knative.dev/v1"
+    "kind" = "Service"
+    "metadata" = {
+      "name" = "helloworld-go"
+      "namespace" = "default"
+    }
+    "spec" = {
+      "template" {
```
Comment (slevenick) [inline mmv1/templates/terraform/examples/cloud_run_anthos.tf.erb:27]: I don't think this works? At least when I try to run this config it complains it is invalid syntax. Have you been able to set this up successfully?

I also have kubernetes-alpha issues, but that's probably unrelated

**Review by slevenick — CHANGES_REQUESTED** [review]
I think we want to also include a guide on this in our guides section. I'm not entirely sure what that would look like, but we probably want to go more in depth into this interaction

File: mmv1/templates/terraform/examples/cloud_run_anthos.tf.erb (hunk 1 above)
Comment (upodroid) [reply mmv1/templates/terraform/examples/cloud_run_anthos.tf.erb:27]: Sorry, there are a few typos in the step but I have tried it now.

**Comment by upodroid** [comment]
Bump

**Comment by upodroid** [comment]
Bump

**Review by slevenick — APPROVED** [review]
Thanks for the contribution! I removed the link to the original issue as I think closing that would require more than just an example, it would likely take the form of a more in-depth walkthrough or guide. I'm not really sure what that would be at this point, but I would like to leave the issue open

**Comment by upodroid** [comment]
That is fine, but that is out of scope as end users are expected to know Kubernetes and Knative to make it work properly.

**Pull request merged**